src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -936,8 +962,11 @@ class as a vector. Label ordering is dictated by the ``classes``
             batch_size = len(deep_features)
 
         y = []
-        for data, in _create_data_iterator(deep_features['deep features'].to_numpy(), None, batch_size=batch_size):
-            y += self._custom_classifier.predict(data).tolist()
+        data_gen = _MXNetDataIterator(deep_features['deep features'].to_numpy(), None, batch_size=batch_size)
+
+        #for data in _create_data_iterator(deep_features['deep features'].to_numpy(), None, batch_size=batch_size):
```

Comment:
Delete this comment out code too?

Reply:
I will if I'm unable to fix the data iterator issue in CustomModel `predict()` within this PR.